app/routes/image-set.js

```diff
@@ -3,6 +3,17 @@ import OrganizationRouteMixin from 'lion-guardians/mixins/organization-route';
 import RequireUserMixin from 'lion-guardians/mixins/require-user';
 
 export default Ember.Route.extend( OrganizationRouteMixin, RequireUserMixin, {
+  beforeModel: function() {
+    var route = this;
+    return this._super().then(function() {
+      if (window.SKIP_MAPS) {
+        return Ember.RSVP.resolve();
+      } else {
+        return route.loadGoogleMap();
```

Comment:
Very interesting. Thanks for the explanation. I'll make the map visible all the time and make marker.isDraggable flag bound to isEditing so you can only move it in editing mode.